lib/kvdb/kvdb_rparams.c

```diff
@@ -417,6 +556,42 @@ dur_mclass_converter(
     return false;
 }
 
+static merr_t
+dur_mclass_stringify(
+    const struct param_spec *const ps,
+    const void *const              value,
+    char *const                    buf,
+    const size_t                   buf_sz,
+    size_t *const                  needed_sz)
+{
+    int         n;
+    const char *param;
+
+    INVARIANT(ps);
+    INVARIANT(value);
+    INVARIANT(buf);
+
+    param = mpool_mclass_to_string[*(enum mpool_mclass *)value];
```

Comment:
Might add `assert(*(enum mpool_class *)value < MD_MED_COUNT)`

Reply:
This cannot ever be possible. That value will always be valid because it couldn't have been set to something invalid.

Reply:
That's exactly what asserts are for -- to prove assumptions you've built into the code.